tests/api_fastapi/core_api/routes/public/test_dag_run.py

```diff
@@ -887,7 +887,7 @@ def test_patch_dag_run(self, test_client, dag_id, run_id, patch_body, response_b
                 {"note": "new_note2", "state": "failed"},
                 200,
             ),
-            ({"update_mask": ["note"]}, {}, {"state": "success", "note": None}, 200),
+            ({"update_mask": ["note"]}, {}, {"state": "success", "note": "test_note"}, 200),
```

Comment:
Same this was updated but is actually a valid test. We should be able to reset the note to `None`

Reply:
You could add another one like:

```
({"update_mask": ["note"]}, {"note":  None}, {"state": "success", "note": "test_note"}, 200),
```

Reply:
The test case will be this right
({"update_mask": ["note"]}, {"note":  None}, {"state": "success", "note": None}, 200)

If the note is in `update_mask` and also the `body` then it will get updated to `None`